backend/gpt/gpt_service.py

```diff
@@ -1,9 +1,19 @@
 from typing import Any, Dict, List
 
 import openai
-from core import env
 from sqlalchemy import Inspector
 
+from core import env
+
+from .flavor_queries import (
```

Comment:
Is there a reason you import all of them, when you have a dict that stores these in `FLAVOR_EXAMPLES`